lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -75,6 +75,17 @@ std::string resolveRelativePathWithinMayaContext(
 MAYAUSD_CORE_PUBLIC
 bool pathAppendPath(std::string& a, const std::string& b);
 
+/**
+ * Appends `b` to the path `a` and returns directory separators as necessary.
```

Comment:
This doesn't really "return directory separators" - it returns a path (by appending two input paths).